tests/main/classic-ubuntu-core-transition-auth/task.yaml

```diff
@@ -24,6 +24,11 @@ execute: |
     echo '{}' > /home/test/.snap/auth.json
 
     echo "Ensure transition is triggered"
+    # wait for steady state or ensure-state-soon will be pointless
+    while snap changes|grep ".*.Doing.*"; do
+        snap changes
```

Comment:
Doing this twice a second will generate a lot of output that will need to be transferred and cached by spread, which is bad on its own, but then on failures it produces those never-ending logs that travis barfs on.

It would be better print nothing, and after N iterations exit 1, and then put the snap changes command in the debug section so that it gets printed on failures. Note that the while above is also printing (grep needs -q).

Same for all cases below.

Reply:
addressed